person.rb

```diff
@@ -0,0 +1,30 @@
+require 'securerandom'
+
+class Person
+  def initialize(name = 'Unknown', age = nil, parent_permission: true)
+    @id = SecureRandom.uuid
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id, :name, :age
```

Comment:
Kindly make the `id` a read-only `attr`. In a real-world application, IDs are used as references and should not be opened to be modified by the user.

Reply:
@brytebee I don't understand what you mean by making the id a read-only attr. It doesn't have any method to reassign it's value, so it should be read-only, right? 
Am I missing something?